README.md

```diff
@@ -2,13 +2,21 @@
 
 A Prometheus exporter for some postgresql metrics.
 
+## Getting Started
+
 You can add as many database connections as you like to the
 `config.yml` file, and run it with:
 
 ```console
 ./postgresql_exporter -config=my/config.yml
 ```
 
+Some stats are hidden from normal database users, so you must grant acess to that:
```

Comment:
Maybe something like its is better:
> By default some stat views like `pg_stat_statements` and `pg_stat_activity` doesn't allow viewing queries run by other users, unless you are a database superuser. Since you probably don't want monitoring to run as a superuser, you can setup a separate monitoring user like this: